implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -223,9 +218,9 @@ private void prepareUdpClient(Publisher<String> publisher) {
                 .host(statsdConfig.host())
                 .port(statsdConfig.port())
                 .handle((in, out) -> out
-                        .options(NettyPipeline.SendOptions::flushOnEach)
                         .sendString(publisher)
                         .neverComplete()
+                        .retry(throwable -> throwable instanceof PortUnreachableException)
```

Comment:
This has the effect that the UdpClient will continue taking items from the Processor and trying to send them, even when it gets the `PortUnreachableException` from trying to send an item. So applications can start up while the StatsD server is down, or if it goes down temporarily, the application doesn't completely stop sending metrics.